codegen/rust/src/pipeline.rs

```diff
@@ -129,7 +129,7 @@ impl<'a> PipelineGenerator<'a> {
 
         let pipeline = quote! {
             pub struct #pipeline_name {
-                #(#table_members),*,
+                #(#table_members,)*
```

Comment:
This change is to handle p4 programs with no tables, which are legit p4 programs.